src/crypto-api.ts

```diff
@@ -40,6 +40,44 @@ export interface GeneratedSecretStorageKey {
     encodedPrivateKey?: string;
 }
 
+/**
+ * Parameter of {@link CryptoApi#bootstrapSecretStorage}
+ */
+export interface CreateSecretStorageOpts {
```

Comment:
please could you put this *after* CryptoApi: it is confusing when the main class in a big file is lost in the middle of the file.